src/main/content/_data/en/en.yml

```diff
@@ -124,4 +124,16 @@ instance:
   app-hub: Application Hub
   oauth:
     configure: Configure OAuth
-  cli-version: "Collection API version: "
\ No newline at end of file
+  cli-version: "Collection API version: "
+  sync-tooltip: Sync collections
+  data-table-headers:
+    collection: Collection
+    version: Version
+    status: Status
+  no-instances: No instance found
+  date-created: Date created
+  modal-confirmation-header: Confirmation
+  modal-content-one: Are you sure you want to deactivate the
+  modal-content-two: "collection?"
```

Comment:
instead of breaking the message into two we can just do `Are you sure you want to deactivate the collection: ` and put the collection name as the last thing. longer unbroken sentences help with translation.